CONTRIBUTING.md

```diff
@@ -87,6 +107,14 @@ Once you downloaded the images you do the following for each screenshot:
   repo, commit it and push it to your PR's branch.
 
 ## Pull Request Reviewers Guide
+If someone requests your review on a pull request,
+read the title and description and assign any other collaborators
+who would want to know about the proposed change.
+
+Decide whether you think that your input is needed,
+and that the PR should wait for your further review before being merged.
+If not, un-assign yourself as a reviewer and leave a comment.
+
```

Comment:
Should we say something like "After a few days have passed without action or comment since a review request was made, progress towards merge of the changes will not require waiting for the requested review to occur. (But alternate reviewers might be need to be identified)" ?

Reply:
That contradicts what I meant to propose, but it does match what we've been doing in practice. To make it explicit, should alternate reviewers or the requesters themselves retract review requests, if they no longer expect a review?

Reply:
My opinion : we are okay if it is clear what the expectations are when a requested review is waiting for action. If your proposed rules are the following, it would be fine with me (and the suggestion I gave above can be ignored)
- PRs will not be merged while outstanding reviews requests require action
- people who are requested to do a review should either do the review within N days, or cancel the review request, or leave a comment on when the review can be expected to be completed.
- when the previous rule is not followed, the requester (or anyone) is free to withdraw/cancel the request
- the requirements of how many / what kinds of reviews must be completed before a PR gets merged will be determined / followed by the person who merges the code. Some manager / merger should comment in the PR when additional reviews must be done before merge.